.eslintrc

```diff
@@ -1,15 +1,8 @@
 {
   "root": true,
   "parser": "@typescript-eslint/parser",
-  "plugins": [
-    "@typescript-eslint",
-    "prettier"
-  ],
-  "extends": [
-    "eslint:recommended",
-    "plugin:@typescript-eslint/recommended",
-    "prettier"
-  ],
+  "plugins": ["@typescript-eslint", "prettier"],
+  "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended", "prettier"],
```

Comment:
This is unexpected.

Reply:
I renamed this to `.eslintrc.json`

Reply:
the ESLint doc suggests the following formats
![image](https://user-images.githubusercontent.com/109317559/213391632-6520b361-d742-4b9d-9555-6f712d20a269.png)